recipes/sfml/all/conanfile.py

```diff
@@ -36,6 +38,24 @@ class SfmlConan(ConanFile):
         "audio": True,
     }
 
+    @property
+    def _min_cppstd(self):
+        # sfml 2.6.0 later requires C++17
+        # https://github.com/SFML/SFML/commit/bd1243891658340c04464f0b3715194ddde5eddf
+        return "17" if Version(self.version) >= "2.6.0" else ""
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "17": {
+                "gcc": "8",
+                "clang": "7",
+                "apple-clang": "12",
+                "Visual Studio": "16",
+                "msvc": "192",
+            },
+        }.get(self._min_cppstd, {})
```

Comment:
SFML 2 only requires C++03 so this should not be necessary. SFML 3 requires C++17 but that has not been released yet.

Reply:
@ChrisThrasher 
Thank you for your comment.
I misunderstand SFML policy.

But I found several files in 2.6.0 use C++11 features.
https://github.com/SFML/SFML/blob/2.6.0/src/SFML/Audio/SoundFileReaderMp3.cpp#L58

I will make `CMAKE_CXX_STANDARD` 11 in `conanfile.py`

Reply:
That's fine. We've tried to maintain C++03 compatibility but apparently some C++11 things have slipped in. Nearly every compiler used these days supports C++11 so this won't be a problem in practice.